tensorflow/python/ops/image_ops_impl.py

```diff
@@ -668,6 +668,11 @@ def rot90(image, k=1, name=None):
   >>> print(a_rot[...,0].numpy())
   [[3 1]
    [4 2]]
+  >>> # rotating `a` clockwise by 180 degrees
```

Comment:
This is already similar to the rotation above. Plus, it's only 90 degrees since that

Reply:
Above example shows counter-clockwise for positive input, I have added the example shows the clockwise output with negative value of k.